hw/ip/rstmgr/doc/_index.md

```diff
@@ -284,31 +284,32 @@ Reset Cause             | Description
 `POR`                   | Cold boot, the system was reset through POR circuitry.
 `LOW_POWER_EXIT`        | Warm boot, the system was reset through low power exit.
 `NDM RESET`             | Warm boot, the system was reset through `rv_dm` non-debug-module request.
+`SW_REQ`                | Warm boot, the system was reset through {{< regref "RST_REQ" >}}.
 `HW_REQ`                | Warm boot, the system was reset through peripheral requests.  There may be multiple such requests.
 
 
 The reset info register is write 1 clear.
 It is software responsibility to clear old reset reasons; the reset manager simply records based on the rules below.
 
 Excluding power on reset, which is always recorded when the device POR circuitry is triggered, the other resets are recorded when authorized by the reset manager.
-Reset manager authorization in turn is based on reset category as indicated by the power manager.
-The power manager observes 3 categories of reset states that are mutually exclusive.
-*   No resets are triggered through the power manager.
-*   Resets triggered by low power entry.
-*   Resets triggered by a peripheral request.
+Reset manager authorization is based on reset categories as indicated by the power manager.
+The power manager observes three categories of reset states that are mutually exclusive.
+*   No reset triggered through the power manager.
+*   Reset triggered by low power entry.
+*   Reset triggered by software or a peripheral request.
 
 Whenever the power manager begins one of the latter two sequences, it sends a hint to the reset manager so that the reset manager can decide which reason to record when the processor reset is observed.
-Whenever the power manager is NOT in one of the two latter states, non-debug-module resets are allowed and directly handled and recorded by the reset manager.
+Whenever the power manager is NOT in one of the two latter states, non-debug-module resets are allowed and directly recorded by the reset manager.
```

Comment:
Here you are explicitly saying the resets not triggered through power manager are non-debug module resets. Perhaps you can make that explicit in that bullet item instead.

Reply:
sure...

Reply:
no that is not what i am saying.